internal/v1/handler.go

```diff
@@ -537,17 +535,22 @@ func (h *Handlers) GetComposes(ctx echo.Context, params GetComposesParams) error
 	ignoreImageTypeStrings := convertIgnoreImageTypeToSlice(params.IgnoreImageTypes)
 
 	// composes in the last 14 days
-	composes, count, err := h.server.db.GetComposes(ctx.Request().Context(), userID.OrgID(), (time.Hour * 24 * 14), limit, offset, ignoreImageTypeStrings)
+	composes, count, err := h.server.db.GetComposes(
+		ctx.Request().Context(),
+		userID.OrgID(),
+		time.Hour*24*14,
+		limit,
+		offset,
+		ignoreImageTypeStrings)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to get composes: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to fetch composes from DB: %w", err)
```

or maybe even composes -> image details?